readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -585,24 +623,44 @@ def _build_latexmk(self, cwd):
             # When ``-f`` is used, latexmk will continue building if it
             # encounters errors. We still receive a failure exit code in this
             # case, but the correct steps should run.
-            '-f',
-            '-dvi-',
-            '-ps-',
-            f'-jobname={self.project.slug}',
-            '-interaction=nonstopmode',
+            "-f",
+            "-dvi-",
+            "-ps-",
+            "-interaction=nonstopmode",
         ]
 
+        if self.project.has_feature(Feature.ENABLE_MULTIPLE_PDFS):
+            cmd.append(f"-jobname={self.project.slug}_%A")
```

Comment:
It would be good to add a comment here explaining what `_%A` means. I guess it will add a counter, or similar.

Reply:
I suppose here we want to use the filename the user has specified in `docs/conf.py`: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-latex_documents (config `targetname`)